### PR TITLE
Add some logs to certificate rotation

### PR DESCRIPTION
**Pull request by bowei**
```release-note
NONE
```

### CODE REVIEW

File: pkg/kubelet/certificate/certificate_manager.go [hunk 1]
```diff
@@ -266,6 +269,8 @@ func (m *manager) shouldRotate() bool {
 }
 
 func (m *manager) rotateCerts() (bool, error) {
+	glog.V(2).Infof("Rotating certificates")
+
```
Comment (Random-Liu) [inline pkg/kubelet/certificate/certificate_manager.go]: Unnecessary empty line.

File: pkg/kubelet/certificate/certificate_manager.go [hunk 2]
```diff
@@ -197,6 +197,7 @@ func (m *manager) Start() {
 	// loop to allow bootstrap scenarios, where the certificate manager
 	// doesn't have a certificate at all yet.
 	if m.shouldRotate() {
+		glog.V(1).Infof("shouldRotate() is true, forcing immediate rotation")
```
Comment (Random-Liu) [inline pkg/kubelet/certificate/certificate_manager.go]: Why `V(1)` higher than others?

**Review by Random-Liu — COMMENTED** [review]
How often does rotation happen? Will this be very spammy?

File: pkg/kubelet/certificate/certificate_manager.go (hunk 2 above)
Comment (bowei) [inline pkg/kubelet/certificate/certificate_manager.go]: this only happens on start up and is fairly important to know about (def: of V(1) log level)

**Comment by bowei** [comment]
rotation takes days, there will not be spammy.

**Comment by bowei** [comment]
@Random-Liu needs /approve no-issue

**Pull request merged**